esmvalcore/cmor/fix.py

```diff
@@ -110,23 +110,23 @@ def fix_metadata(cubes,
         if len(cube_list) != 1:
             cube = None
             for raw_cube in cube_list:
-                if raw_cube.var_name == short_name:
+                if raw_cube.var_name == cmor_name:
```

Comment:
I think this would be the `short_name`, or am I missing something?

Reply:
You are right